algorithms/linfa-linear/benches/ols_bench.rs

```diff
@@ -0,0 +1,28 @@
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use linfa::traits::Fit;
+use linfa_datasets::generate::make_dataset;
+use linfa_linear::LinearRegression;
+use statrs::distribution::{DiscreteUniform, Laplace};
+
+fn perform_ols(num_rows: usize) {
+    let feat_distr = Laplace::new(0.5, 5.).unwrap();
+    let target_distr = DiscreteUniform::new(0, 5).unwrap();
+    let num_feats: usize = 5;
```

Comment:
Can you also run the benchmark with different numbers of features (don't make it too large though).

Reply:
Not a problem, something like 5, 10, 20 sound good? Just want to know what the standard should be.

Reply:
Only really need 5 and 10